vm/actor_interface/src/builtin/account/mod.rs

```diff
@@ -60,19 +60,25 @@ impl State {
     {
         if is_v8_account_cid(&actor.code) {
             return store
-                .get_obj(&actor.state)?
+                .get(&actor.state)?
+                .map(|bz| from_slice(&bz))
+                .transpose()?
```

Comment:
Let's refactor this to a separate function.

Reply:
done